src/sentry/api/endpoints/rpc.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework.exceptions import NotFound, ParseError, PermissionDenied
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import Endpoint, all_silo_endpoint
+from sentry.services.hybrid_cloud.rpc import (
+    RpcArgumentException,
+    RpcResolutionException,
+    RpcSenderCredentials,
+    dispatch_to_local_service,
+)
+
+
+@all_silo_endpoint
```

Comment:
Hmmmmmm....
Ideally, I still believe in having two separate endpoints, for region and control.  The reason is routing related, too.  Being able to say that all control bound rpc endpoints are in a specific subpath makes networking, monitoring, and in general context easier.  Could we extract this as a base class and then have each silo subclass separately?

Reply:
> Hmmmmmm.... Ideally, I still believe in having two separate endpoints, for region and control. The reason is routing related, too. Being able to say that all control bound rpc endpoints are in a specific subpath makes networking, monitoring, and in general context easier. Could we extract this as a base class and then have each silo subclass separately?

I'll follow up on this in a separate branch.